Cargo.toml

```diff
@@ -9,8 +9,7 @@ mp2_v1 = { git = "https://github.com/Lagrange-Labs/mapreduce-plonky2.git", rev =
 parsil = { git = "https://github.com/Lagrange-Labs/mapreduce-plonky2.git", rev = "2.1.0" }
 verifiable-db = { git = "https://github.com/Lagrange-Labs/mapreduce-plonky2.git", rev = "2.1.0" }
 
-alloy = { version = "0.11", default-features = false }
-alloy-primitives = { version = "0.8" }
+alloy = { version = "0.9", default-features = false }
```

Comment:
The PR says that it updates deps; in reality it downgrades them?

Reply:
Yeah, that's ugly – I wanted to sync MR2 & worker alloy, just to err on the side of safety.

Reply:
you can leave alloy out of this PR; I am going to do that separately

Reply:
Will do